packages/edit-site/src/components/page-content-focus-manager/index.js

```diff
@@ -1,21 +1,49 @@
 /**
  * WordPress dependencies
  */
-import { useSelect } from '@wordpress/data';
-
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
 /**
  * Internal dependencies
  */
 import { store as editSiteStore } from '../../store';
 import DisableNonPageContentBlocks from './disable-non-page-content-blocks';
 import EditTemplateNotification from './edit-template-notification';
 import BackToPageNotification from './back-to-page-notification';
+import { unlock } from '../../lock-unlock';
 
 export default function PageContentFocusManager( { contentRef } ) {
-	const hasPageContentFocus = useSelect(
-		( select ) => select( editSiteStore ).hasPageContentFocus(),
+	const { hasPageContentFocus, pageContentFocusType, canvasMode } = useSelect(
+		( select ) => {
+			const { getPageContentFocusType, getCanvasMode } = unlock(
+				select( editSiteStore )
+			);
+			const _canvasMode = getCanvasMode();
+			return {
+				canvasMode: _canvasMode,
+				pageContentFocusType: getPageContentFocusType(),
+				shouldDisplayPageContentTemplates:
+					getCanvasMode() !== 'edit' &&
+					getPageContentFocusType() !== 'disableTemplate',
```

Comment:
Is this being used anywhere? It doesn't appear to be expanded in the above `const { ... } = useSelect` line, so I think these lines can be removed?

Reply:
I left that in there to make sure you were paying attention. :trollface: 

Seriously, thanks for noticing 😄